hazelcast.yaml

```diff
@@ -3,8 +3,6 @@ kind: Hazelcast
 metadata:
   name: hz
 spec:
-  image:
-    tag: 3.11.2
```

Comment:
What about keeping those part or even adding mancenter but use placeholder like `<hazelcast-version>` , `<management-center-version>`... That way users understand that they have to replace it.. Just a suggestion... I am not sure it is a good idea though.

Reply:
I'd keep it as it is. The advantage of this is that you don't need to modify `hazelcast.yaml` to quickly install the Hazelcast resource.